phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/ViewEngines.java

```diff
@@ -90,7 +90,7 @@ public ViewEngines(ThemeManager themeManager, String enginesPath) {
 
         Bindings.bindContent(availableEngines.getTabs(), mappedSubCategoryTabs);
 
-        this.initializeSidebar();
+        this.setSidebar(this.sidebar);
```

Comment:
That's true. This leads us to the question, if we should provide a protected sidebar field, which can be used from the extending classes.